notebooks/notebook_template_review.py

```diff
@@ -268,73 +279,112 @@ def init():
         NotebookRule.costs = ''
     
     @abstractmethod
-    def validate(self, path: str, cells: list) -> None:
+    def validate(self, cells: list) -> None:
```

Comment:
The definition has a `cells` parameter but nothing is passed in when it's called.

Reply:
The subclasses don't conform properly to this abstractmethod.

Reply:
yup, missed that

Reply:
done